fixed sampler name generation

derZ-dev opened the pull request:
- added DPM++ 3M sampler names
- added Exponential scheduler

alexopus commented:
Looks good, but is there really just 'dpmpp_3m' in Comfy? I only see 'dpmpp_3m_sde' in my up-to-date checkout.
Neither does it exist here: https://github.com/civitai/civitai/blob/main/src/server/common/constants.ts#L121-L146

derZ-dev commented:
> Code looks good, but is there really just 'dpmpp_3m' in Comfy? I only see 'dpmpp_3m_sde' in my up-to-date checkout. Neither does it exist here: https://github.com/civitai/civitai/blob/main/src/server/common/constants.ts#L121-L146

No, you're right. Thanks for the hint. I just assumed there was one and therefore added the key-value-pair. Removed it and updated my main branch.

alexopus commented:
Looks good, thanks!
